README.md

```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### ABOUT ME
+My name is Gabrielle Bogert, I am fourth year majoring in Public Policy and minoring in Environmental and Urban Studies. I was born and raised in Miami, the middle child of three girls. 
+**I'm passionate about**
+* American politics
+* Climate change mitigation and adaption
+* Digital media and communication
+* Public transportation (yes I'm a NUMTOT)
+* Libraries
+* Listening to, sharing, and discovering music
+* Scrolling through Twitter and TikTok for hours
+* Going down Wikipedia rabbit holes from the hours of 1 to 4 AM each night of quarantine
+* Reinventing myself constantly by adding hobbies and skills to the institutional memory of my physical form...we're currently on baking
+
+*The last thing that made me laugh is*
+![Funny Tweet](Desktop/hahaha.png)
+
+*The latest thing I've baked is*
+[Earl Grey Yogurt Cake](https://www.bonappetit.com/recipe/earl-grey-yogurt-cake)
+
+
+### PROCESS
+This was my first time using GitHub and while it was a little challenging at first, reading all of the information availible on the CFSS course website helped me immensely. 
+
+**These were my steps to get changes into README.md on GitHub** 
+1. Forked the hw01 repository
+2. Cloned the fork onto my computer by creating a version control project
+3. Made changes to the README.md by adding my biography and homework process
+4. Committed my changes in R Studio
+5. Pushed my changes to GitHub
+6. Made a pull request to submit my homework
+
+I don't know how often I should be committing changes, for this assignment I could do this in one sitting so I only committed once. But let me know if I should be committing more often, based on X amount of lines of code written or X amount of time spent working.
```

Comment:
A good rule of thumb is that you want to commit just as often as you'd save -- basically every time to do some work that you'd consider submit-able. Commiting once for an assignment like this is 100% fine.